plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -79,115 +41,110 @@ export class EventPipelineRunner {
     // KLUDGE: This is a temporary entry point for the pipeline while we transition away from
     // hitting Postgres in the capture endpoint. Eventually the entire pipeline should
     // follow this route and we can rename it to just be `runEventPipeline`.
-    async runLightweightCaptureEndpointEventPipeline(event: PipelineEvent): Promise<EventPipelineResult> {
+    async runLightweightCaptureEndpointEventPipeline(event: PipelineEvent) {
         this.hub.statsd?.increment('kafka_queue.lightweight_capture_endpoint_event_pipeline.start', {
```

Comment:
why don't we change this statsd metric to be `'kafka_queue.event_pipeline.start', { pipeline: 'lightweight_capture' }`